qcodes/instrument/parameter.py

```diff
@@ -288,8 +288,19 @@ def __init__(
         snapshot_exclude: bool = False,
         max_val_age: Optional[float] = None,
         vals: Optional[Validator[Any]] = None,
-        abstract: Optional[bool] = False
+        abstract: Optional[bool] = False,
+        bind_to_instrument: bool = True,
```

Comment:
need to add it to docs (also abstract arg is not in the docs), i gues, to both _Base and normal Parameter?